ws-nextjs-app/pages/[service]/send/[id]/SubmitButton/styles.ts

```diff
@@ -0,0 +1,27 @@
+import { Theme, css } from '@emotion/react';
+
+export default {
+  submit: ({ palette, fontSizes, fontVariants, spacings, mq }: Theme) =>
+    css({
+      backgroundColor: palette.SERVICE_NEUTRAL_CORE,
+      color: palette.WHITE,
+      width: '100%',
+      ...fontVariants.sansBold,
+      ...fontSizes.bodyCopy,
```

Comment:
May be worth just checking with UX on the font size. Looks like it should be 16px from the designs.

Reply:
Maybe outside of this ticket, but I think the `privacyNotice` text needs to be smaller. The designs have it at 14px, whereas it looks like its 16px currently.

Reply:
ahh good spot, will update privacyNotice to use brevier, think the font size is correct for the button tho being bodycopy - based on this:
<img width="392" alt="Screenshot 2024-05-13 at 11 32 55" src="https://github.com/bbc/simorgh/assets/71819756/22c7b036-88e0-4ef1-8a06-fa03c4b03dc3">

Reply:
Cool thanks! Yea maybe its just the privacy notice text throwing my eye off, makes sense. Thanks!